### PR TITLE
docs: add changelogs

### PR DESCRIPTION
**Pull request by gibiw**
Add the changelogs for the following packages:
- qase-pytest
- qase-robotframework
- qase-python-commons

### CODE REVIEW

**Review by n3r — APPROVED** [review]

File: qase-pytest/changelog.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# qase-pytest 6.1.0b1
+
+## What's new
+
+Implement a method which constructs the signature to each test result.
+It takes the file path, the suites, the qase ids and the parameters.
```
Comment (NickVolynkin) [inline qase-pytest/changelog.md:6]:
```suggestion
Implemented a method that constructs a signature for each test result.
It takes the file path, suites, qase IDs, and parameters.
```

File: qase-pytest/changelog.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+# qase-pytest 6.0.3
+
+## What's new
+
+Fixed the issue then specified suite in the decorator was not displayed. Fix [#231]
```
Comment (NickVolynkin) [inline qase-pytest/changelog.md:12]:
```suggestion
Fixed the issue where the suite specified in the decorator was not displayed. Fix [#231]
```

File: qase-pytest/changelog.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+```log
+TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
+```
+
+# qase-pytest 6.0.1
+
+## What's new
+
+Change a name of the complete test run parameter for CLI arguments. Fix [#160]
```
Comment (NickVolynkin) [inline qase-pytest/changelog.md:28]:
```suggestion
Changed the name of the complete test run parameter for CLI arguments. Fix [#160]
```

Comment (NickVolynkin) [reply qase-pytest/changelog.md:28]: what's the old and the new name?

File: qase-pytest/changelog.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+# qase-pytest 6.0.0
+
+## What's new
+
+This is the first release in the 6.0.x series of the Pytest reporter.
```
Comment (NickVolynkin) [inline qase-pytest/changelog.md:34]:
```suggestion
The first release in the 6.0.x series of the Pytest reporter.
```

File: qase-pytest/changelog.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+# qase-pytest 6.0.0b2
+
+## What's new
+
+The sleep profiler was turned off until the test data collection was completed.
```
Comment (NickVolynkin) [inline qase-pytest/changelog.md:40]:
```suggestion
Turn off the sleep profiler until the test data collection is completed.
```

File: qase-python-commons/changelog.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
+# qase-python-commons@3.1.0b4
+
+## What's new
+
+Show the link to the test result in the Qase then the test is failed.
```
Comment (NickVolynkin) [inline qase-python-commons/changelog.md:5]:
```suggestion
Show a link to the test result in the Qase when the test has failed.
```

File: qase-python-commons/changelog.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+    - Update retrieving data from the config in all dependencies.
+
+2. Add a new configuration option `rootSuite` to specify a root suite.
+   This option is available in the config file and the `QASE_ROOT_SUITE` env variable.
+
+# qase-python-commons@3.0.3
+
+## What's new
+
+Fix an issue with getting bool parameter from the config then the parameter's value is `False`
```
Comment (NickVolynkin) [inline qase-python-commons/changelog.md:42]:
```suggestion
Fix an issue with getting bool parameters from the config when the parameter's value is `False`
```

File: qase-python-commons/changelog.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+# qase-python-commons@3.0.2
+
+## What's new
+
+This is the first release in the 3.0.x series of the commons package.
```
Comment (NickVolynkin) [inline qase-python-commons/changelog.md:48]:
```suggestion
First release in the 3.0.x series of the commons package.
```

File: qase-python-commons/changelog.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+
+# qase-python-commons@3.0.2b9
+
+## What's new
+
+Fixed the issues on Windows OS:
```
Comment (NickVolynkin) [inline qase-python-commons/changelog.md:54]:
```suggestion
Fixed the issues on Windows:
```

File: qase-python-commons/changelog.md [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+
+# qase-python-commons@3.0.2b8
+
+## What's new
+
+Fixed the issues:
+
+- return a correct value from the config if the value didn't find
+- mark `Field` model as exported
+- fix the problem when the test run would be completed before the results were sent
```
Comment (NickVolynkin) [inline qase-python-commons/changelog.md:68]:
```suggestion
- Return a correct default value from the config if the value was not found.
- Mark the `Field` model as exported.
- Fix the problem where the test run is completed before the results are sent.
```

File: qase-python-commons/changelog.md [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+
+# qase-python-commons@3.0.2b7
+
+## What's new
+
+Fixed the issue:
```
Comment (NickVolynkin) [inline qase-python-commons/changelog.md:74]:
```suggestion
Fixed the following issue:
```

File: qase-python-commons/changelog.md [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+
+```log
+Creating instance failed: OSError: [ Errno 22 ] Invalid argument: './logs\file.log'
+```
+
+# qase-python-commons@3.0.2b6
+
+## What's new
+
+Fixed the issue:
```
Comment (NickVolynkin) [inline qase-python-commons/changelog.md:84]:
```suggestion
Fixed the following issue:
```

File: qase-python-commons/changelog.md [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+# qase-python-commons@3.0.2b5
+
+## What's new
+
+Fixed the following issues with network profiler:
+
+- ignore requests to `qase.io` host
+- check that `body` and `headers` attributes are in the request
+- fix ignoring response for `urllib3` package
+- add a creation step for redirected requests
```
Comment (NickVolynkin) [inline qase-python-commons/changelog.md:101]:
```suggestion
- Ignore requests to `qase.io` host.
- Check that `body` and `headers` attributes are in the request.
- Fix ignoring response for the `urllib3` package.
- Add a creation step for redirected requests
```

File: qase-python-commons/changelog.md [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+[ ERROR ] Taking listener 'qaseio.robotframework.Listener' into use failed: 
+    Importing listener 'qaseio.robotframework.Listener' failed: 
+       ModuleNotFoundError: No module named 'pkg_resources'
+```
+
+# qase-python-commons@3.0.2b5
+
+## What's new
+
+Fixed the following issues with network profiler:
```
Comment (NickVolynkin) [inline qase-python-commons/changelog.md:96]:
```suggestion
A number of improvements in the network profiler:
```

File: qase-robotframework/changelog.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
+# qase-pytest 3.1.0b1
+
+## What's new
+
+Implement a method which constructs the signature to each test result.
+It takes the file path, the suites, the qase ids and the parameters.
```
Comment (NickVolynkin) [inline qase-robotframework/changelog.md:6]:
```suggestion
Implemented a method that constructs a signature for each test result.
It takes the file path, suites, qase IDs, and parameters.
```

File: qase-robotframework/changelog.md [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+
+# qase-robotframework@3.0.0
+
+## What's new
+
+This is the first release in the 3.0.x series of the Robot Framework reporter.
```
Comment (NickVolynkin) [inline qase-robotframework/changelog.md:12]:
```suggestion
The first release in the 3.0.x series of the Robot Framework reporter.
```

File: qase-robotframework/changelog.md [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+
+## What's new
+
+Fixed the issue:
+
+```log
+AttributeError: 'Result' object has no attribute 'case'
+```
+
+Renamed the internal package name `qaseio` to `qase`
```
Comment (NickVolynkin) [inline qase-robotframework/changelog.md:30]:
```suggestion
Renamed the internal API client package from `qaseio` to `qase`.
```

File: qase-robotframework/changelog.md [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+
+# qase-robotframework@3.0.0b2
+
+## What's new
+
+Fixed an issue with unsupported step status. If the step is `skipped` then mark the step as `blocked`.
```
Comment (NickVolynkin) [inline qase-robotframework/changelog.md:36]:
```suggestion
Fixed an issue with unsupported step status. If the step is `skipped`, mark such a step as `blocked` in the Qase test run.
```

File: qase-robotframework/changelog.md [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+
+## What's new
+
+This is the first release in the 3.0.x series of the Robot Framework reporter.
+
+# qase-robotframework@3.0.0b4
+
+## What's new
+
+The logic for extracting Qase ID has been changed. The reporter will extract the Qase ID after ending test.
```
Comment (NickVolynkin) [inline qase-robotframework/changelog.md:18]:
```suggestion
The reporter will now extract the Qase ID after ending the test.
```

Comment (gibiw) [reply qase-robotframework/changelog.md]: fixed

File: qase-robotframework/changelog.md (hunk 18 above)
Comment (gibiw) [reply qase-robotframework/changelog.md]: fixed

File: qase-robotframework/changelog.md (hunk 15 above)
Comment (gibiw) [reply qase-robotframework/changelog.md]: fixed

File: qase-python-commons/changelog.md (hunk 14 above)
Comment (gibiw) [reply qase-python-commons/changelog.md]: fixed

File: qase-python-commons/changelog.md (hunk 13 above)
Comment (gibiw) [reply qase-python-commons/changelog.md]: fixed

File: qase-python-commons/changelog.md (hunk 12 above)
Comment (gibiw) [reply qase-python-commons/changelog.md]: fixed

File: qase-python-commons/changelog.md (hunk 11 above)
Comment (gibiw) [reply qase-python-commons/changelog.md]: fixed

File: qase-python-commons/changelog.md (hunk 10 above)
Comment (gibiw) [reply qase-python-commons/changelog.md]: fixed

File: qase-python-commons/changelog.md (hunk 9 above)
Comment (gibiw) [reply qase-python-commons/changelog.md]: fixed

File: qase-python-commons/changelog.md (hunk 8 above)
Comment (gibiw) [reply qase-python-commons/changelog.md]: fixed

File: qase-python-commons/changelog.md (hunk 7 above)
Comment (gibiw) [reply qase-python-commons/changelog.md]: fixed

File: qase-python-commons/changelog.md (hunk 6 above)
Comment (gibiw) [reply qase-python-commons/changelog.md]: fixed

File: qase-pytest/changelog.md (hunk 5 above)
Comment (gibiw) [reply qase-pytest/changelog.md]: fixed

File: qase-pytest/changelog.md (hunk 4 above)
Comment (gibiw) [reply qase-pytest/changelog.md]: fixed

File: qase-pytest/changelog.md (hunk 2 above)
Comment (gibiw) [reply qase-pytest/changelog.md]: fixed

File: qase-pytest/changelog.md (hunk 1 above)
Comment (gibiw) [reply qase-pytest/changelog.md]: fixed

File: qase-pytest/changelog.md (hunk 3 above)
Comment (gibiw) [reply qase-pytest/changelog.md]: old name - qase_testops_run_description
new name - qase_testops_run_complete

but it was in the internal logic. For clients, the names have not changed.

**Pull request merged**